src/lib/OpenEXRCore/internal_dwa_classifier.h

```diff
@@ -22,58 +22,67 @@ typedef struct _Classifier
 #define DWA_CLASSIFIER_FALSE 0
 #define DWA_CLASSIFIER_TRUE 1
 
+// clang-format off
+
 static Classifier sDefaultChannelRules[] = {
-    {"R", LOSSY_DCT, EXR_PIXEL_HALF, 0, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"R", LOSSY_DCT, EXR_PIXEL_FLOAT, 0, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"G", LOSSY_DCT, EXR_PIXEL_HALF, 1, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"G", LOSSY_DCT, EXR_PIXEL_FLOAT, 1, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"B", LOSSY_DCT, EXR_PIXEL_HALF, 2, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"B", LOSSY_DCT, EXR_PIXEL_FLOAT, 2, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"Y", LOSSY_DCT, EXR_PIXEL_HALF, -1, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"Y", LOSSY_DCT, EXR_PIXEL_FLOAT, -1, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
-    {"BY", LOSSY_DCT, EXR_PIXEL_HALF, -1, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
+    {"R",  LOSSY_DCT, EXR_PIXEL_HALF,   0, DWA_CLASSIFIER_FALSE, DWA_CLASSIFIER_TRUE},
```

Comment:
here you've made it more readable, which I like. Is clang format going to have a fit?

Reply:
I blocked clang-format in this section with off / on comments, makes it easier to see some times...